src/app.ts

```diff
@@ -917,6 +917,13 @@ router.get(
   safeRequestHandler(businessesAPI.getBusinessAccount)
 );
 
+router.post(
+  "/businesses/:business_id/accounts/:account_id/authorized_persons",
+  middlewares.withBusiness,
+  middlewares.withAccount,
```

Comment:
why do you need this middleware when account is already stored inside the business? this could be useful when you do not have business_id

Reply:
Yes, if we consider the account as a resource of the business you are right, we don't need it.

Reply:
here https://github.com/kontist/mock-solaris/blob/7e4c6b147f82c4651b7607e4615276e0fbfa9b97/src/routes/accounts.ts#L151

Reply:
Yes, thank you, I found it, and it fixed by https://github.com/kontist/mock-solaris/pull/210/commits/cb9a0db67c43d9e43286b8f2c55294b5ce0a3035.